bodhi/tests/server/services/test_updates.py

```diff
@@ -501,6 +501,62 @@ def test_new_update_with_multiple_bugs(self, publish, *args):
         self.assertEqual(up['bugs'][1]['bug_id'], 5678)
         self.assertEqual(up['bugs'][2]['bug_id'], 12345)
 
+    @mock.patch.dict('bodhi.server.validators.config', {'acl_system': u'dummy'})
+    @mock.patch(**mock_valid_requirements)
+    @mock.patch('bodhi.server.notifications.publish')
+    def test_new_update_with_too_low_stable_days(self, publish, *args):
+        update = self.get_update('bodhi-2.0.0-2.fc17')
+        update['stable_days'] = 1
+        r = self.app.post_json('/updates/', update)
+        up = r.json_body
+
+        self.assertEqual(up['stable_days'], 7)
+        self.assertEqual(up['caveats'][0]['description'],
+                         'The number of stable days required was set to the mandatory '
+                         'release value 7 days')
+
+    @mock.patch.dict('bodhi.server.validators.config', {'acl_system': u'dummy'})
+    @mock.patch(**mock_valid_requirements)
+    @mock.patch('bodhi.server.notifications.publish')
+    def test_new_update_with_high_stable_days(self, publish, *args):
+        update = self.get_update('bodhi-2.0.0-2.fc17')
+        update['stable_days'] = 10
+        r = self.app.post_json('/updates/', update)
+        up = r.json_body
+
+        self.assertEqual(up['stable_days'], 10)
+
+    @mock.patch.dict('bodhi.server.validators.config', {'acl_system': u'dummy'})
+    @mock.patch(**mock_valid_requirements)
+    @mock.patch('bodhi.server.notifications.publish')
+    def test_new_update_with_invalid_stable_days(self, publish, *args):
+        update = self.get_update('bodhi-2.0.0-2.fc17')
+        update['stable_days'] = 0
+        r = self.app.post_json('/updates/', update, status=400)
+        up = r.json_body
+
+        self.assertEqual(up['status'], 'error')
+        self.assertEqual(up['errors'][0]['description'], "0 is less than minimum value 1")
```

Comment:
Won't we be setting this to 0 for Rawhide? Maybe we should just ensure it is not negative.